app/styles/ilios-common/components/daily-calendar-event.scss

```diff
@@ -13,6 +14,9 @@
   overflow: hidden;
   z-index: 2;
 
+  &.clickable {
```

Comment:
This shouldn't be necessary since it's defined globally in https://github.com/ilios/common/blob/f6e464e47c35899a655c4cd5b799efa1656a14a5/app/styles/ilios-common/components/body.scss#L82

Reply:
![image](https://user-images.githubusercontent.com/1410427/105777869-0278de80-5f20-11eb-8f74-b9afaeaab7d0.png)

unfortunately, the explicit CSS declaration here is necessary, the `.daily-calendar-event` overwrites `.clickable`. they have the same specificity, but the event class appears after the generic clickable style. 
thinking of alternatives, we could throw a `.non-clickable` style in the class attribute, or perhaps use a different element  that's not a `<button>` for putting those events on the calendar.

Reply:
![image](https://user-images.githubusercontent.com/1410427/105779273-d743be80-5f22-11eb-9b22-c50ed09b9deb.png)

unfortunately, this extra style declaration is necessary. `.clickable` gets overwritten otherwise.

alternatives - none of which i like better: 
- add an explicit `non-clickable` class to the button
- use a `div` instead of a `button`.